requirements.txt

```diff
@@ -1,9 +1,9 @@
-scikit-learn==0.21.3
```

Comment:
I see `requirements.txt` and `conda_requirements.txt` are exactly same. Only difference is `conda_requirements.txt` is not sorted. Can we not have one of them.

Reply:
Yeah, RSMTool consolidated they two requirements file (via work that @jbiggsets did) which is very cool. However, I would like to see how that pans out before making the same change here in SKLL. Let’s file an issue for this and plan for it for v2.1. 

For now, I will also sort `conda_requirements.txt`.